modal/runner.py

```diff
@@ -14,10 +18,10 @@
 from . import _pty
 from ._output import OutputManager, get_app_logs_loop, step_completed, step_progress
 from .app import _LocalApp, is_local
+from .cli.container import _container_exec
```

Comment:
We shouldn't import from `.cli` into here. `.cli` is a "peripheral" module that should import from the core modules. So that means we should move the `_container_exec` function out of it somewhere else.